installer/manifests/common/templates/_images.tpl

```diff
@@ -0,0 +1,39 @@
+{{/*Return the proper serivce image name*/}}
+{{/*{{- include "common.images.image" ( dict "imageRoot" .Values.helmservice.image "global" .Values.global.keptn "defaultTag" .Chart.appVersion) -}}*/}}
+{{- define "common.images.image" -}}
+{{- $registryName := "" -}}
+{{- $repositoryName := .imageRoot.repository -}}
+{{- $tag := include "common.images.tag" (dict "imageRoot" .imageRoot "global" .global "defaultTag" .defaultTag) -}}
+{{- if .global }}
+    {{- if .global.registry }}
+     {{- $registryName = .global.registry -}}
+    {{- end -}}
+{{- end -}}
+{{- if .imageRoot.registry -}}
+  {{- $registryName = .imageRoot.registry -}}
+{{- end -}}
+{{- if $registryName }}
+{{- printf "%s/%s:%s" $registryName $repositoryName $tag -}}
+{{- else -}}
+{{- printf "%s:%s" $repositoryName $tag -}}
+{{- end -}}
+{{- end -}}
+
+{{/*Return the proper serivce image tag*/}}
+{{/*{{- include "common.images.image" ( dict "imageRoot" .Values.helmservice.image "global" .Values.global.keptn "defaultTag" .Chart.appVersion) -}}*/}}
+{{- define "common.images.tag" -}}
+{{- $tag := "" -}}
+{{/* Set Image Tag: if globally set or at service level or use default from Chart.yaml*/}}
+{{- if .global -}}
+    {{- if .global.tag -}}
+       {{- $tag = .global.tag -}}
+    {{- end -}}
+{{- end -}}
+{{- if .imageRoot.tag -}}
+   {{- $tag = .imageRoot.tag -}}
+{{- end -}}
+{{- if not $tag }}
+   {{- $tag = .defaultTag -}}
```

Comment:
@mowies do we need to adapt also the release pipeline? Because the function will default to `.Chart.appVersion` which we set to `latest`

Reply:
During the helm chart build, the `latest` tag is replaced with a proper version tag, so I would say this is fine.
Reference: https://github.com/keptn/keptn/blob/master/gh-actions-scripts/build-helm-charts.sh#L30-L35